src/alacris_lib/side_chain_client.mli

```diff
@@ -20,6 +20,10 @@ val operator_address : Address.t Lazy.t
 (* Address of the configured operator for the current client.
    TODO: actually support more than one operator in the client. *)
 
+val contract_address_client : Address.t Lazy.t
```

Comment:
This comment seems to be copy+pasted from the `operator_address` signature above (so "configured operator for the current client" is a mistake, and of course RE: the TODO you wouldn't have multiple contract addresses) and I think the `_client` suffix is a little confusing.

Reply:
This problem has been corrected. The *_client* has been renamed as *_for_client* which is more accurate I believe.